tests/scripts/all.sh

```diff
@@ -1486,9 +1486,24 @@ component_test_full_no_cipher () {
     scripts/config.py unset MBEDTLS_NIST_KW_C
     scripts/config.py unset MBEDTLS_PKCS12_C
     scripts/config.py unset MBEDTLS_PKCS5_C
-    scripts/config.py unset MBEDTLS_PSA_CRYPTO_C
     scripts/config.py unset MBEDTLS_SSL_TLS_C
     scripts/config.py unset MBEDTLS_SSL_TICKET_C
+    # Disable cipher modes which depends on CIPHER_C
+    scripts/config.py unset MBEDTLS_CIPHER_MODE_CBC
+    scripts/config.py unset MBEDTLS_CIPHER_MODE_CFB
+    scripts/config.py unset MBEDTLS_CIPHER_MODE_OFB
+    scripts/config.py unset MBEDTLS_CIPHER_MODE_CTR
+    # Disable cipher keys/algs which might enable BUILTIN_ALG_ECB_NO_PADDING
+    scripts/config.py unset MBEDTLS_AES_C
+    scripts/config.py unset MBEDTLS_DES_C
+    scripts/config.py unset MBEDTLS_ARIA_C
+    scripts/config.py unset MBEDTLS_CAMELLIA_C
+    scripts/config.py unset MBEDTLS_CHACHAPOLY_C
+    scripts/config.py unset MBEDTLS_CHACHA20_C
+    # Dependencies on AES_C
+    scripts/config.py unset MBEDTLS_CTR_DRBG_C
+    # Disable dependencies on the AEAD algs
+    scripts/config.py unset MBEDTLS_SSL_CONTEXT_SERIALIZATION
     # Indirect dependencies
     scripts/config.py unset MBEDTLS_SSL_CLI_C
     scripts/config.py unset MBEDTLS_PSA_CRYPTO_SE_C
```

Comment:
It should be possible to keep `MBEDTLS_PSA_CRYPTO_SE_C` and `MBEDTLS_PSA_CRYPTO_STORAGE_C` now.